### PR TITLE
Sync CRDs between operator and helm chart

### PR DESCRIPTION
**Pull request by e0ne**
CRDs are synced automatically and CI check is added.

Closes: #134
Signed-off-by: Ivan Kolodiazhnyi <ikolodiazhny@nvidia.com>

### CODE REVIEW

File: scripts/check-manifests.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+# Copyright 2020 NVIDIA
```
Comment (moshe010) [inline scripts/check-manifests.sh:2]: s/2020/2021

Comment (e0ne) [reply scripts/check-manifests.sh:2]: thanks, will fix it once I'll figure out why it doesn't work on CI

**Comment by moshe010** [comment]
see like the CRD are not sync

**Comment by e0ne** [comment]
> see like the CRD are not sync see The command "bash scripts/check-manifests.sh" exited with 1.

I need to make this script more accurate, it checks all files in a repo and I can't reproduce the issue locally

**Comment by e0ne** [comment]
Travis CI failure is not related to the change

File: go.mod [hunk 2]
```diff
@@ -7,6 +7,7 @@ require (
 	github.com/go-logr/logr v0.3.0
 	github.com/googleapis/gnostic v0.5.3 // indirect
 	github.com/k8snetworkplumbingwg/network-attachment-definition-client v1.1.0
+	github.com/mattn/goveralls v0.0.8 // indirect
```
Comment (moshe010) [inline go.mod:10]: why you need goveralls? We don't integrated  Coveralls.io  in network operator at the moment

**Review by moshe010 — CHANGES_REQUESTED** [review]

File: go.mod (hunk 2 above)
Comment (e0ne) [reply go.mod:10]: Need to investigate why Travis CI add it. Let's remove it for now

**Review by moshe010 — APPROVED** [review]

**Pull request merged**